src/app/cube/collection-details/collection-details.component.scss

```diff
@@ -90,26 +110,35 @@ $content-width: calc(#{$max-width} - #{$sidebar-width});
         justify-content: center;
 
         & > *:first-child {
-            margin-right: 0px;
+            margin-right: 0;
         }
     }
 
     .name {
         color: white;
-        font-size: 60px;
+        font-size: 50px;
         text-transform: uppercase;
         font-weight: bold;
-        text-shadow: 0px 0px 25px rgba(0, 0, 0, 0.5);
+        text-shadow: 0 0 25px rgba(0, 0, 0, 0.5);
     }
 
     .long-name {
         color: white;
         font-size: 24px;
-        text-shadow: 0px 0px 25px rgba(0, 0, 0, 0.5);
+        text-shadow: 0 0 25px rgba(0, 0, 0, 0.5);
         font-weight: normal;
     }
 }
 
+.recently-released {
+    font-weight: 200;
+    font-size: 33px;
+    margin-bottom: 25px;
+    text-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
+    color: #333;
```

Comment:
Why is this value hardcoded? Can we replace it with a SCSS variable?

Reply:
Updated.